src/content/en/tools/chrome-devtools/_book.yaml

```
@@ -0,0 +1,20 @@
+upper_tabs:
+- include: /web/_upper_tabs-Fundamentals.yaml
+- name: Tools
+  heading: Chrome DevTools
+  path: /web/tools/
+  menu:
+  - include: /web/tools/_menu_toc.yaml
+  lower_tabs:
+    other:
+    - name: Guides
+      contents:
+      - include: /web/tools/chrome-devtools/_toc.yaml
+    - name: Blog
+      contents:
+      - include: /web/tools/chrome-devtools/blog/_toc.yaml
```

Alternatively, if you want to just tag posts in updates, you could link to there. Then you wouldn't have to create a directory and manage all the content. Plus, you'd get the instant and automatic reach of the readership for /updates/.

```suggestion
      - title: Blog
        path: /web/updates/tags/devtools
```

---

Thanks for the suggestion. I intentional create a separate TOC for that instead of blog because I want the menu and navigation stay consistent in the context. Linking to `/updates` path cause the menu changes.